src/drt/test/ispd18_sample.defok

```diff
@@ -27,6 +27,8 @@ TRACKS X 83800 DO 52 STEP 400 LAYER Metal2 ;
 TRACKS Y 72010 DO 51 STEP 380 LAYER Metal2 ;
 TRACKS X 83800 DO 52 STEP 400 LAYER Metal1 ;
 TRACKS Y 72010 DO 51 STEP 380 LAYER Metal1 ;
+GCELLGRID X 83600 DO 18 STEP 5700 ;
+GCELLGRID Y 71820 DO 16 STEP 5700 ;
```

Comment:
Why is this now necessary?

Reply:
Because now instead of reading guides only by tritonroute, we read guides by fastroute as well which initializes the gcell grid from the given guides.